src/components/postComponent/components/post-button.svelte

```diff
@@ -0,0 +1,46 @@
+<script lang="ts">
+	import { derived } from 'svelte/store';
+	import { isFileSelected, postText } from '$stores';
+
+	// Hilfsfunktion zum Entfernen von Leerzeichen und Zeilenumbrüchen
+	function removeWhitespaceAndNewlines(text: string): string {
+		return text.replace(/\s+/g, '');
+	}
+
+	// Der Button ist aktiviert (klickbar), wenn ein Text (nach dem Entfernen von Leerzeichen)
+	// eingegeben oder eine Datei ausgewählt wurde
+	const inputValid = derived(
+		[isFileSelected, postText],
+		([$isFileSelected, $postText]) =>
+			$isFileSelected || Boolean(removeWhitespaceAndNewlines($postText))
+	);
+</script>
+
+<button
+	type="button"
+	class="btn btn variant-filled-secondary mt-2 buttonPost"
```

Comment:
Hier eventuell primary-color verwenden, dann ist es gleich wie beim Input

Reply:
Ich finde es mit dem secondary schöner, ich denke das können wir uns später nochmals überlegen, ich habe es jetzt mal auf primary gesetzt 👍

Reply:
War nur damit es einheitlich ist :D Weil das ja in der Checkliste stand. Aber mit den Farben müssen wir sowieso nochmal schauen :)